opencti-platform/opencti-front/src/private/components/data/Stream.js

```diff
@@ -58,9 +58,14 @@ class Stream extends Component {
     const { t, theme } = this.props;
     const { sortBy, orderAsc, searchTerm } = this.state;
     const dataColumns = {
+      public: {
```

Comment:
There is a small alignment problem
![image](https://user-images.githubusercontent.com/53513584/214236182-d7c5ac5b-6f4f-495f-9f6e-93e63f60db05.png)

Reply:
Done